promscale/alerts-dashboards-runbooks.md

```diff
@@ -0,0 +1,60 @@
+# Alerts, Dashboards and Runbooks
+
+This sections describes how one can monitor Promscale using dashboards, alerts and
+runbooks to troubleshoot Promscale.
+
+<highlight type="note">
+To use Promscale dashboard make sure to configure Promscale as Prometheus data source 
```

Comment:
We should recommend that people set up a separate Prometheus instance to monitor Promscale. If Promscale goes down it won't be able to alert you that it has gone down. Then they should create a Prometheus data source in Grafana for that Prometheus instance. They also need to ensure the Prometheus instance is scraping the http://<promscale-host>:<promscale-port>/metrics where the default port is 9201